Cooper/Controllers/UsersConnectionController.cs

```diff
@@ -71,45 +72,45 @@ public IActionResult Subscribe(long userId)
 
             string subscriberToken = Request.Headers["Authorization"].ToString().Replace("Bearer ", "");
 
-            UserConnections userConnection = userConnectionService.CreateConnection(userId, subscriberToken);
+            UsersConnection usersConnection = userConnectionService.CreateConnection(userId, subscriberToken);
 
-            bool isSubscribed = userConnectionsRepository.CreateSubscription(userConnection);
+            bool isSubscribed = userConnectionsRepository.CreateSubscription(usersConnection);
                 
             return Ok(isSubscribed);
         }
         
-        [HttpPost("ban/{userId}"), Authorize]
+        [HttpPost("ban/{bannedUserId}"), Authorize]
         [ProducesResponseType(200)]
         [ProducesResponseType(201)]
-        public IActionResult Ban(long userId)
+        public IActionResult Ban(long bannedUserId)
         {
             if (!ModelState.IsValid)
             {
                 return BadRequest(ModelState);
             }
 
-            string subscriberToken = Request.Headers["Authorization"].ToString().Replace("Bearer ", "");
+            string userToken = Request.Headers["Authorization"].ToString().Replace("Bearer ", "");
```

Comment:
I wanted to propose creating an extension method that will return the user from request, but I have already found it in code)))
So just use it) or add the similar one for a token

Reply:
Just added extraa method in AuthExtension for taking token, so logic for creating UsersConnection object by appropriate service is saved